packages/dev/core/src/Meshes/Node/Blocks/conditionBlock.ts

```diff
@@ -57,6 +57,12 @@ export class ConditionBlock extends NodeGeometryBlock {
     })
     public test = ConditionBlockTests.Equal;
 
+    /**
+     * Gets or sets the epsilon value used for comparison
+     */
+    @editableInPropertyPage("Epsilon", PropertyTypeForEdition.Float, "ADVANCED", { embedded: true, notifiers: { rebuild: true } })
+    public epsilon = 0;

```

Comment:
```suggestion
    public epsilon = Epsilon;
```

Reply:
No I need 0 by default to not change the current behavior :)